functions/adjustChatCapacity.ts

```diff
@@ -90,7 +91,9 @@ export const handler: ServerlessFunctionSignature = TokenValidator(
       const { status, message } = await adjustChatCapacity(context, validBody);
 
       return resolve(send(status)({ message, status }));
-    } catch (err) {
+    } catch (err: any) {
+      // eslint-disable-next-line no-console
+      console.error(err);
```

Comment:
This is reintroducing stuff we don't want in

Reply:
Yes but we get rid of most of them [here](https://github.com/techmatters/serverless/pull/289). Let's keep them out please.